src/e2e/java/uk.gov.hmcts.reform.sscs/ProcessFileAndSaveIntoCcd.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.sscs;
+
+import static org.junit.Assert.assertNotNull;
+
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import uk.gov.hmcts.reform.sscs.services.CaseLoaderService;
+
+/**
+ * Class for testing purpose.
+ * When running the CDD dockerized env we can use the method here to confirm
+ * that we can save a Case into the CDD from the CaseLoader successfully.
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ProcessFileAndSaveIntoCcd {
+
+    @Autowired
+    private CaseLoaderService caseLoaderService;
+
+    @Test
+    @Ignore
+    public void givenACase_shouldBeSaveIntoCcd() throws Exception {
```

Comment:
I know this is ignored, but what is the point of this test? It doesn't assert anything? Is there anything we can do to improve this?

Reply:
HI @jackmaloney, it's a e2e test to make sure the caseLoaderService works fine, I mean, if the test passes fine, that is it doesn't throw any execption or issue, then it means the caseLoader app works fine. It's really an e2e test that will be moved, eventually, to the e2e2 project and be owned by the testers.

Reply:
Ok, just wondered if there was any way of verifying within the test that the service actually did what we wanted it to do rather than just rely on an Exception not being thrown. No worries if not

Reply:
don't see  why we should bother about that, really, it's up to the testers to write that up, really.